concourse-production.yml

```diff
@@ -28,3 +28,10 @@ instance_groups:
   azs: [z1]
   networks:
   - name: production-concourse
+- name: iaas-worker
+  vm_type: production-concourse-worker
+  vm_extensions: [production-concourse-iaas-profile]
+  instances: 1
```

Comment:
IDK if we can go smaller than `t2.medium` with all the addons we pack onto those boxes.  `t2.small` only comes with 2GB ram.